ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/StatementExecutorTest.java

```diff
@@ -449,6 +451,65 @@ public void shouldEnforceReferentialIntegrity() {
 
   }
 
+  @Test
+  public void shouldFailCreateAsSelectIfExceedActivePersistentQueriesLimit() {
+    // Given:
+    // Create streams and start two persistent queries
+    createStreamsAndTables();
+    // Prepare to try adding a third
+    final KsqlConfig otherConfig =
+        configWith(KsqlConfig.KSQL_ACTIVE_PERSISTENT_QUERY_LIMIT_CONFIG, 2);
+    final Command csasCommand =
+        givenCommand("CREATE STREAM user2pv AS select * from pageview;", otherConfig);
+    final CommandId csasCommandId =
+        new CommandId(CommandId.Type.STREAM, "_CSASGen2", CommandId.Action.CREATE);
+
+    // When:
+    statementExecutor.handleStatement(csasCommand, csasCommandId, Optional.empty());
+
+    // Then: CSAS statement should fail since exceeds limit of 2 active persistent queries
+    final CommandStatus commandStatus = getCommandStatus(csasCommandId);
+    assertThat(commandStatus.getStatus(), equalTo(CommandStatus.Status.ERROR));
```

Comment:
fyi: `is` can be used in place of `equalTo`, which is less keys to press and takes up less of your line, so you have less code needing to be broken across two lines. It also makes the code _read_ better... 'assert that x is y'...

 Up to you which you use though.